_includes/releases/v22.2/v22.2.0-alpha.1.md

```diff
@@ -0,0 +1,851 @@
+- Clusters that are upgraded to an alpha or other manual build from the development branch will not be able to subsequently upgrade to a release build. [#86345][#86345]
+- Added the `rebalancing.writebytespersecond` and `rebalancing.readbytespersecond` [time series metrics](../{{site.versions["dev"]}}/monitoring-and-alerting.html). These metrics reflect the average number of bytes written and read across all replicas per store, over the last 30 minutes of time. [#80245][#80245]
+
+<h3 id="v22-2-0-alpha-1-command-line-changes">Command-line changes</h3>
+
+- Added support for the `\password` CLI command that enables secure alteration of the password for a user. The given password will always be pre-hashed with the password hash method obtained via the [session variable](../{{site.versions["dev"]}}/set-vars.html) `password-encryption`, e.g., `scram-sha-256` as the default hashing algorithm. [#77975][#77975]
+- Changes the default `debug compact` maximum compaction concurrency to the number of processors, and adds a `--max-concurrency` flag for overriding the new default. [#78987][#78987]
+- The standalone [`cockroach-sql`](../{{site.versions["dev"]}}/cockroach-sql-binary.html) executable now has more compatibility with [`cockroach sql`](../{{site.versions["dev"]}}/cockroach-sql.html), so it can be used as a drop-in replacement. For example, it supports running without a URL, using connection defaults. It also supports overriding `--certs-dir` and other client-side options also supported by `cockroach sql`. [#82020][#82020]
+- `BYTEA` values are now formatted according to the `bytea_output` session setting. [#81943][#81943]
+- The statement tag displayed for `INSERT` statements now has the full information returned by the server. It is the string `"INSERT"` followed by the `OID` of the row that was inserted (which is currently always `0` in CockroachDB), followed by the number of rows inserted. [#81943][#81943]
+- CLI commands that use a SQL connection (e.g., [`cockroach sql`](../{{site.versions["dev"]}}/cockroach-sql.html), [`cockroach node status`](../{{site.versions["dev"]}}/cockroach-node.html)) now support connecting with `PGPASSFILE` and `PGSERVICEFILE`. The behavior is compatible with how `libpq` (the psql C library) behaves. `PGPASSFILE` defaults to `~/.pgpass`. It is a file that contains lines in the format `hostname:port:database:username:password`. The password field from the first line that matches the current connection parameters will be used to connect to the database.  `PGSERVICEFILE` defaults to `~/.pg_service.conf`. It is a file that contains lines in the format `[myservice] host=somehost port=26257 user=someuser`. Any connection parameters (including `passfile` or `password`) can be specified here. Then, a connection string that specifies the `service=myservice` connection parameter will use the values in `PGSERVICEFILE` to connect. [#82389][#82389]
```

Comment:
File formatting and some language:

```
CLI commands that use a SQL connection (e.g., [`cockroach sql`](../{{site.versions["dev"]}}/cockroach-sql.html) and [`cockroach node status`](../{{site.versions["dev"]}}/cockroach-node.html)) now support connecting with `PGPASSFILE` and `PGSERVICEFILE`. The behavior is compatible with how `libpq` (the psql C library) behaves. The `PGPASSFILE` file defaults to the filepath `~/.pgpass`, and has the format `hostname:port:database:username:password`, where the password field from the first line that matches the current connection parameters will be used to connect to the database. The `PGSERVICEFILE` file defaults to the filepath `~/.pg_service.conf`, and has the format:
    ~~~shell
    [myservice]
    host=somehost
    port=26257
    user=someuser
    ~~~
    - Any connection parameters (including `passfile` or `password`) can be specified in this file as well. Then, a connection string that specifies the `service=myservice` connection parameter will use the values in `PGSERVICEFILE` to connect. [#82389][#82389]
 ```

Not sure if I got the final trailing sentences after the codeblock right.